assets/vsphere_problem_detector/12_prometheusrules.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: vsphere-problem-detector
+  namespace: openshift-cluster-storage-operator
+  labels:
+    role: alert-rules
+spec:
+  groups:
+    - name: vsphere-problem-detector.rules
+      rules:
+      - alert: VSphereOpenshiftNodeHealthFail
+        expr:  vsphere_node_check_errors == 1
+        for: 10m
+        labels:
+          severity: warning
+        annotations:
+          message: "VSphere node health checks are failing on {{ $labels.node }} with {{ $labels.check }}"
```

Comment:
This sounds better to me: "VSphere health check {{ $labels.check }} is failing on node {{ $labels.node }}"

But who I am to comment on other's English style :-)

Reply:
You win this round. I renamed. :-)